sympy/physics/wigner.py

```diff
@@ -28,13 +28,14 @@
   Comput. Phys. Commun., Volume 25, pp. 81-85 (1982)
 
 Credits and Copyright
-~~~~~~~~~~~~~~~~~~~~~
+=====================
 
 This code was taken from Sage with the permission of all authors:
 
 https://groups.google.com/forum/#!topic/sage-devel/M4NZdu-7O38
 
-AUTHORS:
+AUTHORS
```

Comment:
The capitalisation here should be same as: https://github.com/sympy/sympy/pull/20413/files#diff-d589b76690102a574278027b9dbf6f19a9918f683058dc955c5962ebd77a8005R174

Reply:
Changed.